cpmpy/expressions/utils.py

```diff
@@ -109,3 +109,13 @@ def eval_comparison(str_op, lhs, rhs):
         raise Exception("Not a known comparison:", str_op)
 
 
+def get_bounds(expr):
+    # can return floats, use floor and ceil when creating an intvar!
+    from cpmpy.expressions.core import Expression
+    if isinstance(expr,Expression):
+        return expr.get_bounds()
+    else:
+        assert is_num(expr), f"All Expressions should have a get_bounds function, `{expr}`"
+        if is_bool(expr):
+            return 0, 1
```

Comment:
This is a known bool, right? Then we can return `0, 0` or `1, 1`, depending on whether the bool is true or false.